shared/js/views.jsx

```diff
@@ -651,6 +654,44 @@ loop.shared.views = (function(_, mozL10n) {
       });
     },
 
+    MIN_CURSOR_DELTA: 3,
+    MIN_CURSOR_INTERVAL: 100,
+    lastCursorTime: 0,
+    lastCursorX: -1,
+    lastCursorY: -1,
+
+    handleMouseMove: function(event) {
+      // Only update every so often.
+      var now = Date.now();
+      if (now - this.lastCursorTime < this.MIN_CURSOR_INTERVAL) {
+        return;
+      }
+      this.lastCursorTime = now;
+
+      var storeState = this.props.cursorStore.getStoreState();
+
+      var deltaX = event.clientX - storeState.videoLetterboxing.left;
+      var deltaY = event.clientY - storeState.videoLetterboxing.top;
+
+      // Skip the update if cursor is out of bounds
+      if (deltaX < 0 || deltaX > storeState.streamVideoWidth ||
+          deltaY < 0 || deltaY > storeState.streamVideoHeight ||
+      // or the cursor didn't move the minimum.
+          (Math.abs(deltaX - this.lastCursorX) < this.MIN_CURSOR_DELTA &&
+           Math.abs(deltaY - this.lastCursorY) < this.MIN_CURSOR_DELTA)) {
+        return;
+      }
+
+      this.lastCursorX = deltaX;
+      this.lastCursorY = deltaY;
+
+      this.props.dispatcher.dispatch(new sharedActions.SendCursorData({
```

Comment:
Should the existing `bootstrap.js` -> `LoopAPI.broadcastPushMessage` -> `CursorPositionChange` be unified to use SendCursorData?